core/net/http-socket/http-socket.h

```diff
@@ -33,6 +33,18 @@
 
 #include "tcp-socket.h"
 
+#ifdef MAX_HTTP_SOCKET_CONF_PATHLEN
+#define MAX_PATHLEN MAX_HTTP_SOCKET_CONF_PATHLEN
+#else
+#define MAX_PATHLEN 80
+#endif
+
+#ifdef MAX_HTTP_SOCKET_CONF_HOSTLEN
+#define MAX_HOSTLEN MAX_HTTP_SOCKET_CONF_HOSTLEN
+#else
+#define MAX_HOSTLEN 40
+#endif
```

Comment:
I wonder about those configuration names.  It seems to me that more appropriate names are `HTTP_SOCKET_CONF_MAX_PATHLEN` and `HTTP_SOCKET_CONF_MAX_HOSTLEN`.